lefthk-core/Cargo.toml

```diff
@@ -15,6 +15,9 @@ thiserror = "1.0.30"
 tokio = { version = "1.14.0", features = ["fs", "io-util", "macros", "net", "rt-multi-thread", "sync", "time"] }
 x11-dl = "2.19.1"
 xdg = "2.4.0"
+ron = "0.8.0"
```

Comment:
ron shouldn't be a dependency of lefthk-core

Reply:
Hm... this would be a little bit problematic because that would mean, that we'd
need to think about a command-to-command-pipe convertion. 

I decided to use `ron` for this, since it's very nice for rust.
Do you have any other suggestions instead?

Reply:
@AethanFoot (ping)

Reply:
Is any of the ron going to be user facing, so is there any point in using it, keep in mind we will have a separate utility, like leftwm does, for handling sending commands

Reply:
> Is any of the ron going to be user facing

well, not really. A user *could* write ron-commands into the command pipe, but
that's it.

> so is there any point in using it

well it simplifies the serialization and deserialization for us when putting the
commands into the command pipe, so I'd say that it does have a point in using
it. Alternatively we can think about another way of sending the commands into
the pipeline.

> keep in mind we will have a separate utility, like leftwm does, for handling sending commands

what do you mean with this? `lefthk` was also using a command-pipe before this
PR.

Reply:
Well we could use something like unix domain sockets, the nice thing with dbus would be that the user could interact with the api through their own means

Reply:
hm... ok, I'm not familiar with `dbus`. Do you recommend any resources to read about it, how it works and how to implement it?

Reply:
(marking as resolved)